src/components/article-body/body-components/image.tsx

```diff
@@ -0,0 +1,13 @@
+export const ArticleImage = ({
+    url,
+    alt,
+}: {
+    url?: string;
+    alt?: string;
+    className?: string;
+}) => {
+    if (!url) {
+        return null;
+    }
+    return <img src={url} alt={alt || ''} />;
```

Comment:
shouldn't we make the alt attribute not optional?

Reply:
The only time you would set it to an empty string is when you don't want the browser to read it as an image

Reply:
https://accessibleweb.com/question-answer/when-should-i-use-a-null-or-empty-alt-tag/